### PR TITLE
#12 Create advantage card component

### LINKED ISSUE #12: Create advantages card component
Specify card component for advantages section:
1. Add beautiful header image.
2. Add main content.

### PR DESCRIPTION
**Pull request by nikolakozaryan**
# Description

Create AdvantageCard, based on Card.

Fixes #12 

## Type of change

Please choose suitable options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Screenshot is attached or not needed

Screenshot:
Screenshot 2022-08-23 193308

### CODE REVIEW

File: src/common/style/abstracts/_variables.scss [hunk 1]
```diff
@@ -7,5 +7,6 @@ $background-color: #E5E5E5;
 
 //FONTS
 $h1-font: 600 46px / 1.15 'Gilroy', sans-serif;
+$h3-font: 600 26px / 1.2 'Gilroy', sans-serif;
```
Comment (OnlyR10) [inline src/common/style/abstracts/_variables.scss]: I suggest using a mixin for fonts

File: src/common/style/abstracts/_variables.scss [hunk 2]
```diff
@@ -7,5 +7,6 @@ $background-color: #E5E5E5;
...
 $paragraph-font: 400 20px / 1.2 'Gilroy', sans-serif;
 $button-font: 500 20px / 1.2 'Gilroy', sans-serif;
```
Comment (OnlyR10) [inline src/common/style/abstracts/_variables.scss]: Please change px to rem

File: src/components/AdvantageCard/AdvantageCard.module.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@import '../../common/style/abstracts/variables';
+
+.title {
+  font: $h3-font;
+  margin: 70px 0 20px;
```
Comment (OnlyR10) [inline src/components/AdvantageCard/AdvantageCard.module.scss]: Please change px to rem

File: src/components/AdvantageCard/AdvantageCard.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Card from '../card/Card';
+import IAdvantageCard from './IAdvantageCard';
+import styles from './AdvantageCard.module.scss';
```
Comment (OnlyR10) [inline src/components/AdvantageCard/AdvantageCard.tsx]: Please change the name of the import from the scss module to classes, as in other PRs

File: src/common/style/abstracts/_variables.scss (hunk 1 above)
Comment (nikolakozaryan) [reply src/common/style/abstracts/_variables.scss]: Mixins are not in the develop branch yet. Will change when all prs will be closed in the separate issue.

File: src/components/AdvantageCard/AdvantageCard.tsx (hunk 4 above)
Comment (nikolakozaryan) [reply src/components/AdvantageCard/AdvantageCard.tsx]: Done

File: src/components/AdvantageCard/AdvantageCard.module.scss (hunk 3 above)
Comment (nikolakozaryan) [reply src/components/AdvantageCard/AdvantageCard.module.scss]: Done

File: src/common/style/abstracts/_variables.scss (hunk 2 above)
Comment (nikolakozaryan) [reply src/common/style/abstracts/_variables.scss]: Done

**Review by YaGrig — APPROVED** [review]

**Comment by OnlyR10** [comment]
It is necessary to pull all the current changes from the develop branch and fix conflicts

**Pull request merged**